mariia-sereda/README.md

```diff
@@ -0,0 +1,72 @@
+# UAKINO.CLUB
+
+Welcome to UAKINO.CLUB! This is the official repository for the [UAKINO.CLUB website](https://uakino.club/), a platform dedicated to Ukrainian cinema enthusiasts. UAKINO.CLUB provides a curated collection of movies, documentaries, and TV shows, allowing users to explore and enjoy the rich cinematic culture.
+
+## Table of Contents
```

Comment:
> Add links to your TOC items so that readers can quickly jump to the section they want.

sorry, i don't quite understand wehere exactly do i need to use table. may you provide more specific and detail description of the problem and what do i need to change?